models/observations.js

```diff
@@ -7,6 +7,8 @@ module.exports = {
     frameworkId: "ObjectId",
     frameworkExternalId: String,
     solutionId: "ObjectId",
+    programId: "ObjectId",
```

Comment:
@aman-tunerlabs - Make this mandatory. Also, add an index to these new fields in DB via migrations